pages.zh/linux/lvs.md

```diff
@@ -0,0 +1,33 @@
+# lvs
+
+> 显示逻辑卷信息.
```

Comment:
```suggestion
> 显示逻辑卷信息。
```

Reply:
Almost done, Thank you @NealSCarffery .

Reply:
I'll apply this quick fix to get merged the first PR of @NealSCarffery :raised_hands: